integration-tests/test_motd.py

```diff
@@ -0,0 +1,52 @@
+import contextlib
+import os
+
+import pytest
+
+
+DOT_REGISTERED_PATH = "/etc/insights-client/.registered"
+DOT_UNREGISTERED_PATH = "/etc/insights-client/.unregistered"
+
+MOTD_PATH = "/etc/motd.d/insights-client"
+MOTD_SRC = "/etc/insights-client/insights-client.motd"
+
+
+pytestmark = pytest.mark.usefixtures("register_subman")
+
+
+@pytest.fixture
+def clean_system():
+    """Clean up the system before every test."""
```

Comment:
maybe give this fixture a better name & comment on what it does, as IMHO `clean_system` a bit too generic